spec/suites/autocomplete.ts

```diff
@@ -448,5 +499,25 @@ export function autocompleteSuite({
 
             assignMock.mockClear()
         })
+
+        it("should call search when keyqord is submitted with keyboard, with isKeyword=true", async () => {
+            const user = userEvent.setup()
+
+            await waitFor(() => handleAutocomplete(render()))
+
+            await user.type(screen.getByTestId("input"), "black")
+
+            await waitFor(async () => {
+                await user.keyboard("{arrowdown}")
+                await user.keyboard("{arrowdown}")
+                await user.keyboard("{enter}")
+
+                expect(searchSpy).toHaveBeenCalledWith(expect.anything(), {
+                    track: "serp",
+                    redirect: false,
+                    isKeyword: true,
+                })
+            })
+        })
     })
```

Comment:
Add a test with click event?

Reply:
`should call search on keyword click with isKeyword=true` this test is with a click event.